src/scripts/site.js

```diff
@@ -397,10 +392,6 @@ var AppInfo = {};
             define("registerElement", ["document-register-element"], returnFirstDependency);
         }
 
-        define("imageFetcher", [componentsPath + "/images/imageFetcher"], returnFirstDependency);
-
-        var preferNativeAlerts = browser.tv;
```

Comment:
why is this gone?

Reply:
imageFetcher was removed, since it was pretty much integrated into imageLoader.

preferNativeAlerts was a leftover from another PR that moved the castSender, dialog and confirm modules out of site.js.